Samples/iOS-Swift/iOS-Swift/Tools/SentryUIApplication.h

```diff
@@ -0,0 +1,58 @@
+@class UIApplication;
+@class UIScene;
+@class UIWindow;
+@class UIViewController;
+@protocol UIApplicationDelegate;
+
+typedef NS_ENUM(NSInteger, UIApplicationState);
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * A helper tool to retrieve informations from the application instance.
+ */
+@interface SentryUIApplication : NSObject
```

Comment:
`m`: Why do we need this header file here? Please add a short comment with the explanation or remove it.

Reply:
If it is needed here for some reason, we should use the one already in /Sources. See how SentryDevice.h/.mm are included in iOS-Swift/tvOS-Swift.

Reply:
Adding the comment.
This is exposing private API for testing.

I rather have a copy of the header instead of using the same .h file because we have a better control of what is being exposed, and any change in the `#imports` in the original file wont affect the sample.